tests/acceptance/index.js

```diff
@@ -31,8 +31,10 @@ async function runTests (browsers) {
     if (numberTestsFailed > 0) {
       process.exit(1);
     }
-  }
-  finally {
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  } finally {
     await testcafe.close();
```

Comment:
does anything weird happen if we exit the process without closing testcafe?

Reply:
not that I know of but, thinking about it further, I think I should close it before forcefully exiting